pkg/services/sqlstore/migrations/temp_user.go

```diff
@@ -44,4 +49,65 @@ func addTempUserMigrations(mg *Migrator) {
 		{Name: "status", Type: DB_Varchar, Length: 20},
 		{Name: "remote_addr", Type: DB_Varchar, Length: 255, Nullable: true},
 	}))
+
+	tempUserV2 := Table{
+		Name: "temp_user",
+		Columns: []*Column{
+			{Name: "id", Type: DB_BigInt, IsPrimaryKey: true, IsAutoIncrement: true},
+			{Name: "org_id", Type: DB_BigInt, Nullable: false},
+			{Name: "version", Type: DB_Int, Nullable: false},
+			{Name: "email", Type: DB_NVarchar, Length: 190},
+			{Name: "name", Type: DB_NVarchar, Length: 255, Nullable: true},
+			{Name: "role", Type: DB_NVarchar, Length: 20, Nullable: true},
+			{Name: "code", Type: DB_NVarchar, Length: 190},
+			{Name: "status", Type: DB_Varchar, Length: 20},
+			{Name: "invited_by_user_id", Type: DB_BigInt, Nullable: true},
+			{Name: "email_sent", Type: DB_Bool},
+			{Name: "email_sent_on", Type: DB_DateTime, Nullable: true},
+			{Name: "remote_addr", Type: DB_Varchar, Length: 255, Nullable: true},
+			{Name: "created", Type: DB_Int, Default: "0", Nullable: false},
+			{Name: "updated", Type: DB_Int, Default: "0", Nullable: false},
+		},
+		Indices: []*Index{
+			{Cols: []string{"email"}, Type: IndexType},
+			{Cols: []string{"org_id"}, Type: IndexType},
+			{Cols: []string{"code"}, Type: IndexType},
+			{Cols: []string{"status"}, Type: IndexType},
+		},
+	}
+
+	addTableReplaceMigrations(mg, tempUserV1, tempUserV2, 2, map[string]string{
+		"id":                 "id",
+		"org_id":             "org_id",
+		"version":            "version",
+		"email":              "email",
+		"name":               "name",
+		"role":               "role",
+		"code":               "code",
+		"status":             "status",
+		"invited_by_user_id": "invited_by_user_id",
+		"email_sent":         "email_sent",
+		"email_sent_on":      "email_sent_on",
+		"remote_addr":        "remote_addr",
+	})
+
+	// Ensure outstanding invites are given a valid lifetime post-migration
+	mg.AddMigration("Set created for temp users that will otherwise prematurely expire", &SetCreatedForOutstandingInvites{})
+}
+
+type SetCreatedForOutstandingInvites struct {
+	MigrationBase
+}
+
+func (m *SetCreatedForOutstandingInvites) Sql(dialect Dialect) string {
+	return "code migration"
+}
+
+func (m *SetCreatedForOutstandingInvites) Exec(sess *xorm.Session, mg *Migrator) error {
+	created := time.Now().Unix()
```

Comment:
Yeah that's a good point - might be strange to have a date in the future, but at least this will ensure everyone will some reasonable amount of time in the off chance any open invites are migrated

Reply:
> might be strange to have a date in the future

That's very true. Didn't think about this. So when setting created to "now" it will expire after 24 hours, per default. I suggest to revert that then. Sorry for the confusion.